lookbook/docs/patterns/15-colors.md.erb

```diff
@@ -68,6 +68,151 @@ The following variables of Primer are overwritten with the `--accent-color`:
 
 When implementing colors in the application, we should ensure compliance with Primer's [accessibility standards](https://primer.style/foundations/color/accessibility).
 
+## Attributes colors
```

Comment:
Please explain what you mean with `Attribute colours`. Otherwise nobody understands that your are talking about Type or Status and why it is important to change them